django_nanoid/models.py

```diff
@@ -34,6 +35,7 @@ def deconstruct(self):
         kwargs['secure_generated'] = self.secure_generated
         kwargs['alphabetically'] = self.alphabetically
         kwargs['size'] = self.size
+        kwargs['prefix'] = self.size
```

Comment:
Could you @lodye509 please change this to  `kwargs['prefix'] = self.self.prefix` ?

Reply:
oops! thank you for catching, now updated.

> Could you @lodye509 please change this to `kwargs['prefix'] = self.prefix` ?